Vagrantfile

```diff
@@ -22,7 +23,25 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   # Create a forwarded port mapping which allows access to a specific port
   # within the machine from a port on the host machine. In the example below,
   # accessing "localhost:8080" will access port 80 on the guest machine.
-  # config.vm.network "forwarded_port", guest: 80, host: 8080
+  config.vm.network "forwarded_port", guest: 3000, host: 3000
+  config.omnibus.chef_version = :latest
+  config.vm.provision "chef_zero" do |chef|
+    chef.cookbooks_path = 'cookbooks'
+    chef.add_recipe 'parkmap'
+
+    chef.json = {
+        "ruby" => {
+            "version" => "2.2.0"
+        },
+        "postgresql" => {
+            "version" => "9.4",
```

Comment:
[rubocop] Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.